src/microbe_stage/systems/MicrobeAISystem.cs

```diff
@@ -787,10 +799,11 @@ private void FleeFromPredators(ref WorldPosition position, ref MicrobeAI ai, ref
                 // If the predator is right on top of us there's a chance to try and swing with a pilus
                 ai.MoveWithRandomTurn(2.5f, 3.0f, position.Position, ref control, speciesActivity, random);
             }
+        }
 
-            // Sprint until full strain
+        // Sprint until full strain if fears enough
+        if (RollCheck(speciesFear, Constants.MAX_SPECIES_FEAR, random))
```

Comment:
```suggestion
        if (speciesFear > Constants.MAX_SPECIES_FEAR / 2.0f)
```

It's confusing, but I would suggest a raw check on the value rather than a roll check. The RollCheck function re-rolls 3 times a second every time the AI runs, which means any microbe would eventually sprint, whereas a threshold check is constant, meaning some species will choose a sprinting strategy and other will select a "slow and steady" route.

Reply:
Actually, I had another idea, and per your description of the PR I went ahead and pushed it.